content/en/continuous_integration/quality_gates/_index.md

```diff
@@ -0,0 +1,164 @@
+---
+title: Quality Gates
+kind: documentation
+description: Learn how to use Quality Gates in your pipeline.
+is_beta: true
+further_reading:
+- link: "/continuous_integration/tests"
+  tag: "Documentation"
+  text: "Learn about Test Visibility"
+---
+
+## Overview
+
+{{< callout url="#" header="false" >}}
```

Comment:
@alai97 Do you know why this is different from what we have for [static analysis](https://github.com/DataDog/documentation/blob/master/content/en/continuous_integration/static_analysis/_index.md?plain=1#L17-L21)?

Reply:
Also we will want to add a Google Form link for the private beta form, right @KickingTheTV?

Reply:
http://dtdg.co/code-quality

Reply:
```suggestion
{{< callout url="http://dtdg.co/code-quality" header="false" >}}
```